kimchi/src/folding/mod.rs

```diff
@@ -22,13 +22,28 @@ type ScalarField<C> = <<C as FoldingConfig>::Curve as AffineCurve>::ScalarField;
 
 pub trait FoldingConfig: Clone + Debug + Eq + Hash + 'static {
     type Column: FoldingColumnTrait + Debug + Eq + Hash;
+
+    /// The type of challenge that the Sponge returns
```

Comment:
This is not completely correct, it is the type of an abstract challenge that can be found in the expressions you provide as your constraints, but it does not represent an actual value like you can get from the sponge.

Reply:
Thanks for your clarification. I guess a full test would help visualize what each part corresponds to. Right now I commented out a preliminary test inside `mod.rs` to check one single constraint, circuits of just one row. But there's still a lot to parse -> initialize the srs, domain, commit to the witnesses, fold the instances, pass the constraints, and finally prove it. Are you planning to write any test of this type?